### ISSUE TITLE
Increased differentiation of Mycobacterium avium subspecies?

### ISSUE THREAD

**Issue by Dx-wmc** [jodyphelan/NTM-Profiler#41]
Hi, I've been doing research on Mycobacterium avium recently but the subspecies differentiation is giving me a headache, your tool I tried but couldn't differentiate, will this be able to add differentiation of subspecies in the future?

**Comment by jodyphelan** [comment]
Yes this is definitely on the horizon. My plan is to include a SNP-barcode for subspecies prediction (similar how we do it for MTBC). Is there a particular subspecies you are interested in?

**Comment by Dx-wmc** [comment]
An excellent piece of good news, I can't wait to use it.
